cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/TableSink.java

```diff
@@ -61,7 +62,14 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   @Override
   public void initialize(BatchSinkContext context) throws Exception {
     super.initialize(context);
-    recordPutTransformer = new RecordPutTransformer(tableSinkConfig.getRowField(),
+    Schema outputSchema = null;
```

Comment:
If outputSchema is null, all the fields from the StructuredRecord are 'put' into Put. This was the behavior earlier. The old tests already test for this. The test that I have added also tests again for the case of outputSchema being null.